packages/protocol/contracts/governance/Governance.sol

```diff
@@ -679,7 +698,12 @@ contract Governance is
 
     address account = getAccounts().voteSignerToAccount(msg.sender);
     uint256 totalLockedGold = getLockedGold().getAccountTotalLockedGold(account);
-    _vote(proposal, proposalId, index, totalLockedGold, account, voteValues, weights);
+
+    require(
+      totalLockedGold >= yesVotes.add(noVotes).add(abstainVotes),
+      "Voter doesn't have enough locked Celo (formerly known as Celo Gold) "
+    );
```

Comment:
This would probably save gas, but don't know if it is really useful to check, but you can also verify the same requirement that you used before
```
   uint256 votingPower = yesVotes.add(noVotes).add(abstainVotes);
   require(votingPower > 0, "Empty vote"); // or something similar
```

Reply:
Currently this particular scenario can be used for revoking votes of one proposal, I would probably like to keep this